src/pages/_app.tsx

```diff
@@ -56,13 +56,16 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
   useEffect(() => {
     // Displays the Zendesk widget only if user is signed in and on the home page
     if (!window.zE) return;
-    if (!signedIn || Boolean(componentSrc?.componentAccountId && componentSrc?.componentName)) {
+    if (
+      (!signedIn || Boolean(componentSrc?.componentAccountId && componentSrc?.componentName)) &&
+      !router.route.startsWith('/founders')
```

Comment:
This lines of codes makes Zendesk widget visible on `/founders` page even if the user is not logged in